tests/hypothesis/test_prf.py

```diff
@@ -0,0 +1,101 @@
+# This file is dual licensed under the terms of the Apache License, Version
+# 2.0, and the BSD License. See the LICENSE file in the root of this repository
+# for complete details.
+from __future__ import absolute_import, division, print_function
```

Comment:
nitpick: Perhaps a newline between the license header and the `__future__` to stay consistent with the rest of the project?